docs/provider/provider-setup.md

```diff
@@ -144,8 +144,8 @@ lavad tx pairing stake-provider "{NETWORK_NAME}" \
 Param description (and examples):
 - `NETWORK_NAME` - The ID of the serviced chain, see [how to query the full list](provider#chains). Example `COS4` or `FTM250`
 - `STAKE_AMOUNT` - The amount you are willing to stake for being a provider for the specific network. Example `2010ulava`
-- `SERVICED_NODE_IP` - External IP of the node that will service the requests. Example `51.92.133.253` (to find your public IP, run `curl ifconfig.me`)  
-- `SERVICED_NODE_PORT` - Port of the node that will service requests. Example `19921`
+- `LISTEN_IP` - External IP of the node that will service the requests. Example `51.92.133.253` (to find your public IP, run `curl ifconfig.me`)  
```

Comment:
I would rephrase this to 
an IP that the provider exposes to listen to incoming requests from consumers - or something close to it 

same in the port below 

as they both have no relation to the node that will service requests a provider is not a node :)

Reply:
@ranlavanet  WDYT about `exposed_ip`, `exposed_port` ?

Reply:
i suggest ```exposed_listener_ip```

Reply:
I would remove IP altogether and use something like PROVIDER_HOST instead, to encourage the use of CNAMEs / DNS entries instead of IPs (which have the potential to change). Of course IPs should be supported too

Reply:
Thanks. 
@kobrien good point.

@ranlavanet @omerlavanet as part of the upcoming redesign, does this makes sense to you to use a host and not separate IP?

Reply:
"all IP addresses everywhere on our repo can be host names", not to be overly pedantic but I dont think that is self-evident, whereas with "HOST:PORT" i would assume that could either be an IP or hostname. If you prefer to stick with IP:PORT it would be good to comment in the docs that IP can also be a hostname/fqdn :)